packages/components/pie-checkbox/src/index.ts

```diff
@@ -51,7 +53,7 @@ export class PieCheckbox extends FormControlMixin(RtlMixin(LitElement)) implemen
     public aria: CheckboxProps['aria'];
 
     @query('input[type="checkbox"]')
-    private checkbox?: HTMLInputElement;
+    public checkbox!: HTMLInputElement;
```

Comment:
I made it public to help with the tests, but I'm happy to change it back if we like, it's not super important.

Regarding the `!`, I think it's a safe assumption that the input element will always exist, otherwise something has gone extremely wrong!